hieradata_aws/class/staging/mongo.yaml

```diff
@@ -9,3 +9,14 @@ govuk_env_sync::tasks:
     temppath: "/var/lib/mongodb/.dumps"
     url: "govuk-staging-database-backups"
     path: "mongo-api"
+govuk_env_sync::tasks:
```

Comment:
This doesn't need a second `govuk_env_sync::tasks:` key - it's already on line 1.